test/yaml/Makefile

```diff
@@ -34,6 +34,15 @@ iperf3-vcl-image:
 	done
 	docker build -t calicovpp/test-iperf3-vcl -f ./iperf3-vcl/Dockerfile ./iperf3-vcl
 
+.PHONY: sidecar-vcl
+sidecar-vcl:
+	for f in libvcl_ldpreload.so libvppcom.so.24.10.0 libvlibmemoryclient.so.24.10.0 \
+	        libsvm.so.24.10.0 libvppinfra.so.24.10.0 libvlibapi.so.24.10.0 ; do \
+		cp ${VPP_DIR}/build-root/build-vpp-native/vpp/lib/x86_64-linux-gnu/$${f} ./sidecar-vcl/ ; \
+	done
+	cp ${VPP_DIR}/extras/libmemif/build/lib/libmemif.so ./sidecar-vcl/
+	docker build -t calicovpp/vclsidecar -f ./sidecar-vcl/Dockerfile ./sidecar-vcl
+
```

Comment:
Can we make this part of the `make image` and `make push` directives of
`vpp-manager/Makefile` ? That way we have CI taking care of it.

Also we should probably not depend on the VPP version i.e. something like :
```make
	for pkg in vpp vpp-plugin-core vpp-plugin-dpdk libvppinfra vpp-dbg ; do \
		cp vpp_build/build-root/$$pkg_*.deb $(IMAGE_DIR) ; \
	done
```